app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
> I wonder if emitting the signin.success metric before the user gets to the ready.js view might be a better place? 

That has the same problem as I mentioned about regarding about:accounts - that code is only run after the poll completes. If the poll never starts, then that won't be run.

Since we know a subset of users will either close the browser tab or never poll in the first place, for users that must confirm their email, I wonder if instead of logging a "signin.success" in the originating window we instead log "signin.success" in the _verification_ window in [complete_sign_up.js->_notifyBrokerAndComplete](https://github.com/mozilla/fxa-content-server/blob/11179349b2ff997eaa6eaef894c92296fc7cb11a/app/scripts/views/complete_sign_up.js#L126) would give us a good approximation of successful signins. In that function we can call `this.isSignIn()`, and if that returns true, then log a `signin.success`. We know the event won't have been logged in the tab in which the user signed in, but a message *will* have been sent to the browser. If a user opens the verification link and successfully verifies, that could be considered a successful sign in.